### PR TITLE
Clarify class-based helper format documentation.

### PR DESCRIPTION
**Pull request by cantsin**
Fixes https://github.com/emberjs/guides/issues/1318

Also-by: Dig <dig.vargas@locusenergy.com>

### CODE REVIEW

File: source/localizable/templates/writing-helpers.md [hunk 1]
```diff
@@ -239,26 +239,9 @@ should export a subclass of [`Ember.Helper`][1]. Helper classes must contain a
...
-    } else {
-      return 'Not logged in';
-    }
-  }
-});
-```
-
-In fact, we could also refactor the above stateless helper into a class-based
-helper by making the function into a `compute` method on the class:
+In fact, we could also refactor the above `format-currency` helper
```
Comment (mixonic) [inline source/localizable/templates/writing-helpers.md]: How about:

> An an exercise, here is the above `format-concurrency` helper refactored into a class-based helper:

File: source/localizable/templates/writing-helpers.md [hunk 2]
```diff
@@ -274,9 +257,27 @@ export default Ember.Helper.extend({
 });
 ```
 
-This is exactly equivalent to the example above. You can think of the
-function version as a shorthand for the longer class form if it does not
-require any state.
+This is exactly equivalent to the `format-currency` example above. You
+can think of the function version as a shorthand for the longer class
+form if it does not require any state.
```
Comment (mixonic) [inline source/localizable/templates/writing-helpers.md]: ~~state~~. "if the helper does not require dependency injection."

**Comment by cantsin** [comment]
@mixonic updated, thank you for the suggestions!

**Comment by acorncom** [comment]
@homu r+  Thanks, @cantsin!

**Pull request merged**